src/pybind/rados/rados.pyx

```diff
@@ -889,6 +889,13 @@ Rados object in state %s." % self.state)
         if ret >= 0:
             return int(ret)
         elif ret == -errno.ENOENT:
+            rev = self.wait_for_latest_osdmap() 
```

Comment:
@jdurgin  Oh,thank you  I understand it, also does it  need a para  when we do not find the pool.Then we get the latest osdmap to check. The para default is false. when user set it to true then we check the pool in the latest osdmap. how about it?